PWGDQ/DataModel/ReducedInfoTables.h

```diff
@@ -477,6 +477,13 @@ DECLARE_SOA_TABLE(V0Bits, "AOD", "V0BITS", //!
 // iterators
 using V0Bit = V0Bits::iterator;
 
+namespace DalBits
+{
+DECLARE_SOA_COLUMN(DALITZBits, dalitzBits, int); //!
```

Comment:
The type here should be unsigned. I suggest we limit to uint8_t, similar to the v0Bits table